Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/details/SubmissionDetails.tsx

```diff
@@ -60,12 +57,10 @@ const SubmissionDetails = () => {
 
     useEffect(() => {
         if (isNil(currentSubmission)) {
-            return;
+            // return;
         }
 
-        const { problem: { id } } = currentSubmission;
-
-        loadContestByProblemId(id);
+        // loadContestByProblemId(id);
```

Comment:
Please leave comments in the code, explaining why this useEffect becomes useless... and on other places where it's not clear

Reply:
Yes it will be deleted in the PR for this issue https://github.com/SoftUni-Internal/exam-systems-issues/issues/937

Reply:
Ok, just put the same comment in the code with link to the issue. It is better to have it in the code, where it is much easier to see.
We should make sure v2 is kept clean as much as possible, or at least well documented, where it is not as clean..
Even if we plan to fix it soon, this is not guaranteed 100%.

Reply:
Fixed